Sources/BraintreeCore/BTGraphQLHTTP.h

```diff
@@ -23,6 +23,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface BTGraphQLHTTP : BTHTTP
 
+- (void)handleRequestCompletion:(nullable NSData *)data
+                       response:(nullable NSURLResponse *)response
+                          error:(nullable NSError *)error
+                completionBlock:(void (^)(BTJSON * _Nonnull, NSHTTPURLResponse * _Nonnull, NSError * _Nonnull))completionBlock;
+
```

Comment:
There is not an easy way to test this new behavior as `OHHTTPStubs` doesn't let you send nil data and automatically converts it to an empty data object. Adding this to the internal header allows us to unit test this update. I explored overriding the `HTTPStubsResponse` but that also requires eventually overriding pieces of `Foundation` due to how `HTTPStubsResponse` works under the hood.

Reply:
Nice!